src/class-parsely.php

```diff
@@ -798,6 +799,17 @@ public function print_dynamic_tracking_note() {
 	 * Actually inserts the code for the <meta name='parsely-page'> parameter within the <head></head> tag.
 	 */
 	public function insert_parsely_page() {
+		/**
+		 * Filters whether the Parse.ly meta tags should be inserted in the page.
+		 *
+		 * By default, the tags are inserted.
+		 *
+		 * @since 2.6.1
+		 */
```

Comment:
```suggestion
		 * @since 2.6.1
		 *
		 * @param bool $insert_parsely_page
		 */
```

FWIW, I think this "parsely_page" identity isn't great - we should be talking about "meta tags" or "structured data". We need a better collective name for these two metadata formats, that can then be applied to filter names etc. e.g. `wp_parsely_should_insert_metadata`.